### Evolutionary comparisons [Dueck 2015](https://doi.org/10.1002/bies.201500124)
Based on the neutral theory of molecular evolution, something that is evolutionarily important will tend to be conserved between species. So if single cell variation in expression of a particular gene is important, we expect it to be conserved between species.
- Problem: say we get a value of 10 in mice and 12 in rats for some variability metric, how do we know if this is significantly different? Since we don't really have a negative control for a gene we know variation isn't important for.
- [Dueck 2015](https://doi.org/10.1186/s13059-015-0683-4) did this and found evidence for conserved variation in neuronal genes between mice and rats

### Use non-specificity of siRNAs to determine effects of general variability [Dueck 2015](https://doi.org/10.1002/bies.201500124)
siRNAs are usually used to target specific mRNAs but we know that they have significant off-target effects. We could design an siRNA against a random sequence not present in the genome of the species of interest so it should theoretically have random non-specific effects targeting random genes. Ideally, this would be transfected using lentiviruses under regulated promoter control e.g. a tet-on promoter so that firs we would measure phentypic things and then again after "activating the siRNA" plus measuring the transcriptome to see what the siRNA actually targeted

### miRNAs to tune expression of individual genes